docs/sources/panels-visualizations/visualizations/bar-gauge/index.md

```diff
@@ -103,6 +103,12 @@ Limit the minimum height of the bar row in the horizontal direction.
 
 Automatically show y-axis scrollbar when there is a large amount of data.
 
+### Max height
+
+Limit the maximum height of the bar row in the horizontal direction.
+
+Automatically show y-axis scrollbar when there is a large amount of data.
```

Comment:
```suggestion
Limit the maximum height of the bar row when the gauge is oriented horizontally.

Automatically show y-axis scrollbar when there's a large amount of data.
```

Reply:
@imatwawana sure, no problem, I'll update it, we didn't merge [this PR](https://github.com/grafana/grafana/pull/76042) yet but as soon as we merge it, I'll update this one 🙂